src/main/java/com/google/common/base/Stopwatch.java

```diff
@@ -0,0 +1,244 @@
+/*
```

Comment:
is it possible to have 3rd party code like this under `src/vendor` instead of `src/main` to indicate it is from a 3rd party (similar to how projects use lib/vendor)?